index.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: default
+title: home
+---
+
+<span style="font-size:1.25em; font-weight:bold">What is this?</span> A weekly
+club for reviewing Bitcoin Core PRs in the Freenode `#bitcoin-core-pr-reviews`
+channel at **18:00 UTC on Wednesdays**.
+
+<span style="font-size:1.25em; font-weight:bold">What's it for?</span> To
+help newer contributors learn about the Bitcoin Core review process. Review Club
+is *not* primarily intended to help open PRs get merged (although that might be
+a nice side-effect). The Review Club will give participants the tools and
```

Comment:
This seems to be a frequent misunderstanding. People often think that Review Club is to focus review on a PR to help it gets merged, much like the "high-priority for review" list. That's not a goal of review club, which I want to make explicit here.

Reply:
Oh, interesting. I could see a potential incidental effect of the additional activity inciting experienced reviewers to look at a PR sooner than they might otherwise have, but it was clear to me that it's not a primary goal of the club, and I reckoned review weight for maintainers is more about quality than quantity anyway; the [peer review
process](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review) makes that clear.